src/app/shared/models/steam/ssmt.ts

```diff
@@ -105,6 +105,21 @@ export interface GeneralSteamOperations {
 export interface Modification {
     ssmt?: SSMT;
     exploreOpportunities?: boolean;
+    exploreOppsShowOperationsData?: SavingsOpportunity,
+    exploreOppsShowUnitCosts?: SavingsOpportunity,
+    exploreOppsShowBoilerData?: SavingsOpportunity,
+    exploreOppsShowCondensateHandling?: SavingsOpportunity,
+    exploreOppsShowHeatLoss?: SavingsOpportunity,
+    exploreOppsShowSteamUsage?: SavingsOpportunity,
+    exploreOppsShowCondensingTurbine?: SavingsOpportunity,
+    exploreOppsShowHighToLowPressureTurbine?: SavingsOpportunity
+    exploreOppsShowHighToMediumPressureTurbine?: SavingsOpportunity
+    exploreOppsShowMediumToLowPressureTurbine?: SavingsOpportunity
+}
+
+export interface SavingsOpportunity {
```

Comment:
I believe you defined this interface in another place. Probably should used a shared one unless there is something unique about it from assessment type to assessment type

Reply:
Everything else looks good